lib/widgets/header.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:flutter/material.dart';
+import 'package:movie_db/constants/constants.dart';
+
+class MovieHeader extends StatelessWidget {
+  const MovieHeader({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        Padding(
+          padding: const EdgeInsets.all(Constants.infoPadding),
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.center,
+            children: [
+              Stack(
+                children: <Widget>[
+                  // Stroked text as border.
```

Comment:
This type of comment is not necessary to add.

Reply:
Fixed.